### ISSUE TITLE
apt -y update fails in image jrottenberg/ffmpeg:4.1-nvidia, jrottenberg/ffmpeg:4.2-nvidia and jrottenberg/ffmpeg:4.2.3-nvidia

### ISSUE THREAD

**Issue by nafisatd** [jrottenberg/ffmpeg#247]
Hey.

I am trying to build a docker image using the docker images jrottenberg/ffmpeg:4.1-nvidia , jrottenberg/ffmpeg:4.2-nvidia and jrottenberg/ffmpeg:4.2.3-nvidia as base images. When I run apt -y update, the update fails giving the following message:

```
debconf: unable to initialize frontend: Dialog
debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 76.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.26.1 /usr/local/share/perl/5.26.1 /usr/lib/x86_64-linux-gnu/perl5/5.26 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.26 /usr/share/perl/5.26 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
debconf: falling back to frontend: Teletype

LD_LIBRARY_PATH contains the traditional /lib directory,
but not the multiarch directory /lib/x86_64-linux-gnu.
It is not safe to upgrade the C library in this situation;
please remove the /lib/directory from LD_LIBRARY_PATH and
try again.

dpkg: error processing archive /var/cache/apt/archives/libc6_2.27-3ubuntu1.2_amd64.deb (--unpack):
 new libc6:amd64 package pre-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 /var/cache/apt/archives/libc6_2.27-3ubuntu1.2_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)

```

Can you please let me know the possible reasons for this occurrence.
Thank you

**Comment by jrottenberg** [comment]
yeah it's new , a bit frustrating because that was introduced silently from a previous PR, might add a small tasks to verify, apt-get, yum, apk continues to work on the final image... I'll look into it.

**Comment by nuqz** [comment]
> w , a bit frustrating because that was introduced silently from a previous PR, might add a small tasks to verify, apt-get, yum, apk continues to work on the final image... I'll look into it.

@jrottenberg got this error today. It seems it is related to recent `libc` update. I've removed `/lib` from `LD_LIBRARY_PATH` as was said in the message, updated `libc`, then added back the `/lib` dir to `LD_LIBRARY_PATH`, it seems it worked.

And also... I would like to take this opportunity to thank you for this image, it helped us a lot. It would be really great if there was also a version that includes chromaprint capabilities.

**Comment by jrottenberg** [comment]
Yeah @GnaphronG extraordinaire jumped on it with https://github.com/jrottenberg/ffmpeg/pull/248

Hopefully I can merge it today.

Thank you for the support, it's been fun overall, not sure about https://github.com/acoustid/chromaprint / https://www.ffmpeg.org/ffmpeg-formats.html#chromaprint-1 but if you make an issue for it, we can pick it up (I didn't know about that one)

**Comment by jrottenberg** [comment]
Ok azure devops was quick for that build, it passes, give a try to the new image @nafisatd, and close if it works for you.

**Comment by nafisatd** [comment]
Hey @jrottenberg 
The issue is resolved for jrottenberg/ffmpeg:4.1-nvidia and jrottenberg/ffmpeg:4.2-nvidia.
There is no image named jrottenberg/ffmpeg:4.2.3-nvidia. My bad!
The issue still persists in jrottenberg/ffmpeg:4.2.3-nvidia1804.
I haven't mentioned that in the problem statement. So I consider this issue closed.
Thanks alot for your prompt response. :)
